index.html

```diff
@@ -0,0 +1,162 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone Project 1</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav>
+        <div class="mobile-version">
+            <div class="hamburger"><img src="images/images/mobile_menu 1.png" alt="loading"></div>
+            <div class="logo"><img src="images/images/logo.png" alt=""></div>
+            <div class="links links-inactive">
+                <span class="close">&times</span>
+                <ul>
+                    <li><a href="" class="nav-link">About</a></li>
+                    <li><a href="" class="nav-link">Program</a></li>
+                    <li><a href="" class="nav-link">Join</a></li>
+                    <li><a href="" class="nav-link">Sponsor</a></li>
+                    <li><a href="" class="nav-link">News</a></li>
+                </ul>
+            </div>
+        </div>
+        <div class="desktop-version">
+            <ul id="first-nav">
+                <li><a href="#facebook"><img src="images/images/Rectangle.png" alt=""></a></li>
+                <li><a href="#twitter"><img src="images/images/twitter (1).png" alt=""></a></li>
+                <li class="eng">English</li>
+                <li class="eng">My page</li>
+                <li class="eng">Logout</li>
+            </ul>
+            <ul id="second-nav">
+                <li><div class="desk-logo"><img src="images/images/logo.png" alt=""></div></li>
+                <li><a href="" class="nav-link">About</a></li>
+                <li><a href="" class="nav-link">Program</a></li>
+                <li><a href="" class="nav-link">Join</a></li>
+                <li><a href="" class="nav-link">Sponsor</a></li>
+                <li><a href="" class="nav-link">News</a></li>
+                <li class="navbar-link campaign">YLS Campaign</li>
+            </ul>
+        </div>
```

Comment:
- You have done great so far :+1: :100: But there are some requirements you still need to meet before moving to the next milestone. :+1: 
- [ ] As stated in the project requirement, the menu items need to be aligned by the right of the page. I'll suggest you adjust your CSS to match the provided design. :+1: 

- - Screenshot:
![Selection_013](https://github.com/denigit5/First-Capstone/assets/107632780/4820c8b2-88e3-4c1f-b691-5703a62d68a3)

- [ ] Also following the best practices, the logo is meant to be wrapped in an anchor tag `a` and not in a div, the logo should be in an anchor tag with a link to the home page. I'll suggest you update this. :+1: :+1:

Reply:
As suggested by the previous code reviewer please kindly corrected this issue. This is a requested change and it need to be addressed.